docs/releases/updating_ez_platform.md

```diff
@@ -48,6 +48,8 @@ or
 git pull ezplatform-ee <version>
 ```
 
+> Don't forget the `v` here, you want to pull the tag `<version>` and not the branch `<version>` (i.e: `v1.11` and not `1.11`)
```

Comment:
The example should say `v1.11.0` since version tags are normally triplets. In addition to the `v` this also helps separating them from branches.

Good PR, thanks!

Reply:
It would be also cool to format this as a "Pretty Tip Box"(tm) (like we have at the bottom of the front page: http://doc.ezplatform.com/en/master/). This is the way to do it:

```
!!! tip

    Don't forget the `v` here...
```